src/components/BaseComponents/BaseModal.tsx

```diff
@@ -37,12 +41,24 @@ export default function BaseModal({
                             leaveFrom="opacity-100 scale-100"
                             leaveTo="opacity-0 scale-95"
                         >
-                            <Dialog.Panel className={`min-h-[430px] w-full overflow-hidden rounded-2xl bg-white p-2 text-left shadow-xl transition-all ${dialogClass}`}>
+                            <Dialog.Panel
+                                className={`min-h-[430px] w-full overflow-hidden rounded-2xl bg-white p-2 text-left shadow-xl transition-all ${dialogClass}`}
+                            >
                                 <Dialog.Description as="section">
                                     {children}
-                                    <div className="absolute right-3 top-3">
+                                    <div className="absolute right-3 top-3 flex gap-2">
```

Comment:
```suggestion
                                    <div className="absolute right-3 top-3 flex gap-2 items-center">
```
Probably the two icons being identical in height won't create any misalignment but better be safe than sorry.

Reply:
sure, updated